apps/prairielearn/assets/scripts/instructorQuestionSettingsClient.ts

```diff
@@ -11,6 +19,46 @@ onDocumentReady(() => {
   );
   const saveButton = document.querySelector<HTMLButtonElement>('#save-button');
 
+  new TomSelect('#topic', {
+    valueField: 'name',
+    searchField: ['name'],
+    closeAfterSelect: true,
+    items: [(document.querySelector('#topic') as HTMLSelectElement).dataset.selectedTopic],
+    plugins: ['dropdown_input'],
+    maxItems: 1,
+    placeholder: 'Select a topic',
```

Comment:
Nit, let's specify this in the markup (`placeholder="..."`) instead.

Reply:
Moved the `placeholder` to the markup in [98f2dd5](https://github.com/PrairieLearn/PrairieLearn/pull/10868/commits/98f2dd558c3f286228530e2a7899b8db990394be).